app/markdown-it-plugins/cite.js

```diff
@@ -0,0 +1,177 @@
+/* eslint no-param-reassign: 0, yoda: 0, no-continue: 0 */
+import Bibtex from '../bibtex';
+
+
+module.exports = (md) => {
+  const defaultRender = md.renderer.rules.fence;
+  const bibtex = new Bibtex();
+
+  // override fence renderer for `cite`
+  md.renderer.rules.fence = (tokens, idx, options, env, self) => {
+    const token = tokens[idx];
+
+    if ('cite' === token.info) {
+      const entries = bibtex.parse(md.utils.escapeHtml(token.content));
```

Comment:
Note for myself: maybe I could avoid to parse the content twice (here and below in the fence block parser)